qiskit_ibm_provider/accounts/management.py

```diff
@@ -24,11 +24,8 @@
 )
 _DEFAULT_ACCOUNT_NAME = "default"
 _DEFAULT_ACCOUNT_NAME_LEGACY = "default-legacy"
-_DEFAULT_ACCOUNT_NAME_CLOUD = "default-cloud"
 _DEFAULT_ACCOUNT_NAME_IBM_QUANTUM = "default-ibm-quantum"
-_DEFAULT_ACCOUNT_NAME_IBM_CLOUD = "default-ibm-cloud"
-_DEFAULT_CHANNEL_TYPE: ChannelType = "ibm_cloud"
-_CHANNEL_TYPES = [_DEFAULT_CHANNEL_TYPE, "ibm_quantum"]
+_CHANNEL_TYPES = ["ibm_quantum"]
```

Comment:
Since `qiskit-ibm-runtime` will depend on `qiskit-ibm-provider`, we should make the account management code more generic, so `qiskit-ibm-runtime` can reuse it. For example, if you move these global variables into the `AccountManager` class, then `qiskit-ibm-runtime` can inherit and overwrite.